pkg/ccl/backupccl/backupbase/settings.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Licensed as a CockroachDB Enterprise file under the Cockroach Community
+// License (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     https://github.com/cockroachdb/cockroach/blob/master/licenses/CCL.txt
+
+package backupbase
+
+import (
+	"github.com/cockroachdb/cockroach/pkg/settings"
+	"github.com/cockroachdb/cockroach/pkg/util"
+)
+
+// TODO(adityamaru): Move all backup related cluster settings to this file.
```

Comment:
Is this a goal?

If a setting is only used during destination resolution, or only used during data exporting, etc, moving it here is the opposite of dependency hygiene; now if we change this setting, or add another one, etc, we're needlessly adding churn to a dependency of other packages, because something specific to one package leaked out into a common package.

Reply:
makes sense, I needed this setting in a place that I could import into `testutils.go` so that it is registered at the time the test cluster is started. I think that place can be `backupinfo` when it is introduced. Regardless, I'll remove this TODO.